and-bible/app/src/main/java/net/bible/service/device/ProgressNotificationManager.kt

```diff
@@ -118,7 +118,6 @@ class ProgressNotificationManager {
         val builder = NotificationCompat.Builder(app, DOWNLOAD_NOTIFICATIONS_CHANNEL)
```

Comment:
Maybe we should also make the channel name generic if it is not just used for downloads.

Reply:
True. Changed.